lambda/variables.tf

```diff
@@ -122,3 +123,15 @@ variable "filename" {
   description = "Allows a filename to be passed directly to the module instead of using the generic ones"
   default     = ""
 }
+
+variable "use_image" {
```

Comment:
Do you need a separate variable here. Can you add a local variable like `use_image = var.image_url != ''` and then reference `local.use_image` 
What do you think? It reduces the number of variables and stops you getting in a weird situation where you've set use_image to true but haven't provided an image_url.

Reply:
I don't like using string checking for conditions, I think booleans are cleaner